romeo/src/system.rs

```diff
@@ -152,16 +156,99 @@ async fn deploy_asset_contract(config: &Config, client: LockedClient) -> Event {
     Event::AssetContractCreated(txid)
 }
 
+async fn mint_asset(config: &Config, client: LockedClient, deposit_info: DepositInfo) -> Event {
+    let tx_auth = TransactionAuth::Standard(
+        TransactionSpendingCondition::new_singlesig_p2pkh(config.stacks_public_key()).unwrap(),
+    );
+
+    let recipient = PrincipalData::Standard(StandardPrincipalData(
+        deposit_info.recipient.version,
+        *deposit_info.recipient.bytes.as_bytes(),
+    ));
+
+    let function_args = vec![
```

Comment:
This code will most likely need to be in a function. But assume we can just leave this for when you add the burn logic.